### DISCUSSION TITLE
After update, more than 20 devices are missing

### DISCUSSION THREAD

**Discussion by Micware2** [Koenkk/zigbee2mqtt#11734] [Devices]
raspberry pi 4
Zigbee stick/chip CC2652
Raspbian GNU/Linux 10 (buster)

Zigbee2MQTT version   :  **1.19.1** commit: 9bd46932
Coordinator type      : zStack3x0
Coordinator revission : 20201026
Frondend version      : 0.4.16 

**Updated to** 

Zigbee2MQTT version : **1.24.0** commit: 7a2ddf24
Frondend version    : 0.6.76

GUI shows in sttings End devices: 18 and Router: 5 

After following the procedure (described in the link below)
[link to update documentation](https://www.zigbee2mqtt.io/guide/installation/01_linux.html#for-later-update-zigbee2mqtt-to-the-latest-version)

I have more than 50 devices listed in "/opt/zigbee2mqtt/data/configuration.yaml" but only 23 of them are listed in the "Devices" view in the GUI of zigbee2mqtt

I don't know much about the inner workings of Zigbee2mqtt but I assume that the devices must also be on the zigbee stick.
is there a way to check that (which devices are on my USB stick)?

This is the 2nd time I update Zigbee2MQTT since using it (about 1 year now). And the first time I had the same problem. I solved that by adding the missing devices again, but that was a lot of work.

I'm trying to understand where things are going wrong and what I could possibly do differently with an upgrade/update
How do I get my devices back without adding(paring) them back one by one?

Are the versions too far apart to upgrade or has there been a breaking change?

For the rest I am satisfied/happy with it zigbee2mqtt , thanks for making it.

All help, tips and technical explanations are welcome. 
I am an IT technician by profession and should therefore be able to understand it a little. ;-)

**Comment by lux73** [comment]
at least [v1.21.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.21.0) has a breaking Change with paired Devices via Routers...

so i could image that various Devices that are missing now was connected via Routers before

in my Opinion it is not recommended to waiting so long for Updates

**Reply by Micware2** [reply]
Thank you for your quick response.
Due to circumstances there has been quite some time between the update of different versions.

Does it make any difference to the range, if I pair everything via the coordinator instead of a zigbee router devices that is closer to the endpoint. does it still use the mesh functionality to have a good reach with all endpoints
According to the specifications of the CC2652, more than 100 devices can be paired with this stick.

If I understand correctly, there is no other option than to re-pair the devices that are now missing with the Zigbee stick again

**Comment by Micware2** [comment]
I can't find an option to only pair devices with the coordinator. does that mean that a device always pairs with the nearest zigbee router or can I tell zigbee2mqtt that it can only pair with the coordinator?
